README.md

```diff
@@ -11,6 +11,7 @@ https://top90.io
 ## Development Environment Setup
 1. Install Go
 2. Create a `.env` in the root directory and add the contents of `.env.sample` to it with real values
+3. Create `/keys` in the root directory and add the cert for DB tunneling
```

Comment:
The cert file currently needs to be called `defaultec2.pem` since it is hardcoded to that [here](https://github.com/wweitzel/top90-backend/blob/289a52b7b24d3b115320c8ce0534e17dbc6e80cc/Makefile#L52). We should probably state that name here

Reply:
Addressed in [4506674](https://github.com/wweitzel/top90-backend/pull/1/commits/4506674e1eee71a47eb423bd4746bb4f3e3a07a8)